app/assets/stylesheets/admin/pages/_structure.scss

```
@@ -1,12 +1,11 @@
 .admin_structures, .admin_structures_locales, .admin_structures_administratives {
-  $grid-columns:      6;
+  $grid-columns: 6;
   $grid-gutter-width: 1.25rem;
   @import "../../bootstrap_grid";
 
-  .without_sidebar {
-    .structure {
-      @include make-col(3);
-    }
+  // positionne le main content de la page structure au même niveau avec ou sans sidebar
```

Je chipote sur le commentaire, mais j'aurais écrit : 
```suggestion
  // positionne le #main_content de la page structure au même niveau avec ou sans sidebar
```